src/app.ts

```diff
@@ -17,4 +17,10 @@ app.use(express.urlencoded({ extended: false }));
 
 app.use('/', router);
 
+// tu zmienic nizej, p
 
+const mongoose = require('mongoose');
```

Comment:
use module export/imports: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

remove this line:
`const mongoose = require('mongoose');`
and add:
`import mongoose from 'mongoose';`
at the top of the file (